--- dhcp-4.4.2/common/Makefile	2021-04-05 07:22:55.477979272 +0000
+++ dhcp-4.4.2/common/Makefile	2021-04-05 07:28:19.384015841 +0000
@@ -284,7 +284,7 @@ BINDSRCDIR = /my_docker_fs/gitRepositori
 BINDSUBDIR = ${top_srcdir}/bind
 CC = gcc
 CCDEPMODE = depmode=gcc3
-CFLAGS = -g -O2  -Wall -fno-strict-aliasing -I$(top_srcdir)/includes -I/my_docker_fs/gitRepositories/EmbeddedACCompanion/dhcp-4.4.2/bind/include
+CFLAGS = -g -O2  -Wall -Werror -fno-strict-aliasing -I$(top_srcdir)/includes -I/my_docker_fs/gitRepositories/EmbeddedACCompanion/dhcp-4.4.2/bind/include
 CPP = gcc -E
 CPPFLAGS = 
 CYGPATH_W = echo
